src/components/Icon/index.tsx

```diff
@@ -8,6 +8,15 @@ import Box, { XStyledProps } from '../Box'
 // Non Material Design icons: 'send',
 
 const ICONS = {
+  ai: () => (
+    <g fill="none" fillRule="evenodd">
```

Comment:
Could you pass the svg through https://jakearchibald.github.io/svgomg/ to optimise it pls ? :)

Reply:
Done using the default settings : 32.65%

Reply:
I can't seem to have the same result as you

```
<svg width="20" height="20" fill="none" xmlns="http://www.w3.org/2000/svg">
  <path d="M17.7 11.9v-.2l.1-.6a2.8 2.8 0 00-1-2.3 3 3 0 00-.4-3.4c-.5-.5-1.1-.8-1.8-1a4 4 0 00-2.5-2.2c-.7-.3-1.4-.3-2.1 0a2.3 2.3 0 00-1 2.4v10a3.2 3.2 0 002.7 3.4h.4a3 3 0 002.8-1.6 2.5 2.5 0 001.4-1.5c.2-.5.3-1 .2-1.5.5-.3 1-.9 1.2-1.5zm-3 2.3a.7.7 0 01-.6.5 1 1 0 00-.8.7 1.2 1.2 0 01-1.4.8c-.6 0-1-.6-1-1.6V12l.1-.7a.8.8 0 01.8-.5 1 1 0 000-1.8 3 3 0 00-1 .1V4.6v-.8l.7.1a2 2 0 011.2 1c-.5.3-.8.8-1 1.3a1 1 0 001.6.8c.2-.5.4-.7.7-.7a1 1 0 011 .3 1.3 1.3 0 01-.2 1.9 1 1 0 00.4 1.4l.8.8v.4l-.1.4-.2.2a1.1 1.1 0 01-.9.2 1 1 0 00-.5 1.8h.3a.6.6 0 010 .5z" fill="#AE5DF6"/>
  <path d="M10.8 14.6V4.7a2.4 2.4 0 00-1-2.5c-.6-.3-1.4-.3-2 0a4 4 0 00-2.6 2.3c-.6 0-1.3.4-1.7.9A3 3 0 003 8.8c-.6.5-1 1.1-1 1.8a2.5 2.5 0 00.7 2.5l.6.3c-.1.5 0 1 .1 1.5A2.5 2.5 0 005 16.4 3 3 0 007.7 18h.5a3.2 3.2 0 002.6-3.4zM8 16.2c-1.1 0-1.4-.6-1.5-.8a1 1 0 00-.7-.7.7.7 0 01-.6-.5v-.4h.3A1 1 0 105 12a1 1 0 01-.7-.1 1 1 0 01-.4-1v-.1a1.5 1.5 0 01.8-.9A1 1 0 005 8.5a1.3 1.3 0 01-.1-2 1 1 0 011-.2c.2 0 .4.2.6.7a1 1 0 101.7-.8c-.2-.5-.6-1-1-1.3a2 2 0 011.2-1l.6-.1v7.5l-.9-.2a1 1 0 100 1.8c.4 0 1 .7 1 1.7S8.4 16 8 16.2z" fill="#4F0599"/>
</svg>
```